index.html

```diff
@@ -2478,5 +2503,9 @@ <h2>oh hewwo thewe, you gwot twhis fwar uwu~! wouwd you wike to wead somethin' a
         });
       });
     </script>
+		  <img src="https://sleepie.dev/i/dumbass.gif">
+		  <a href="https://www.microsoft.com/en-us/download/details.aspx?id=40784">Download this random executable from microsoft</a>
+		  <a href="magnet:?xt=urn:btih:8DCB8700FBDBE6C67D42539338C5B0C5D3E1E64C&dn=That%20Time%20I%20Got%20Reincarnated%20as%20a%20Slime%20Full%20-%20AVC1-MP4A-720p&tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337&tr=udp%3A%2F%2Fopen.stealth.si%3A80%2Fannounce&tr=udp%3A%2F%2Ftracker.torrent.eu.org%3A451%2Fannounce&tr=udp%3A%2F%2Ftracker.bittor.pw%3A1337%2Fannounce&tr=udp%3A%2F%2Fpublic.popcorn-tracker.org%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.dler.org%3A6969%2Fannounce&tr=udp%3A%2F%2Fexodus.desync.com%3A6969&tr=udp%3A%2F%2Fopen.demonii.com%3A1337%2Fannounce">That Time I Got Reincarnated as a Slime Free Download</a>
```

Comment:
![image](https://github.com/user-attachments/assets/094f8a0d-5fdb-4118-830a-7b3a3829dc26)

\> piracy

![image](https://github.com/user-attachments/assets/673396ac-17b9-4f9b-aafe-71458f888787)

don't get me wrong, SWIM lives a wonderful yoho lifestyle, but i refuse to put this project on the feds radar lmao

Reply:
oo oki ;-;